src/code/z_sound_source.c

```diff
@@ -1,19 +1,20 @@
+#include "z64sound_source.h"
 #include "global.h"
 
 void SoundSource_InitAll(PlayState* play) {
     SoundSource* sources = &play->soundSources[0];
     s32 i;
 
     // clang-format off
-    for (i = 0; i < ARRAY_COUNT(play->soundSources); i++) { sources[i].countdown = 0; }
+    for (i = 0; i < SOUND_SOURCE_COUNT; i++) { sources[i].countdown = 0; }
     // clang-format on
```

Comment:
This this is worth a fake label?

Reply:
Do we use fake labels for this kind of stuff? I can't remember of any tbh